package.json

```diff
@@ -13,14 +13,18 @@
     "android-clean": "cd android && ./gradlew clean && cd ..",
     "ios-clean": "rm -rf ios/Pods && rm -rf ios/build",
     "pod-install": "cd ios && pod install && cd ..",
-    "clean-install": "watchman watch-del-all && rm -rf $TMPDIR/react-native-packager-cache-* && rm -rf $TMPDIR/metro-bundler-cache-* && rm -rf node_modules/ && yarn cache clean && yarn install && yarn android-clean && yarn ios-clean && yarn pod-install",
+    "clean-install": "watchman watch-del-all && rm -rf $TMPDIR/react-native-packager-cache-* && [ -d $TMPDIR/metro-bundler-cache-* ] && rm -rf $TMPDIR/metro-bundler-cache-* || true && rm -rf node_modules/ && yarn cache clean && yarn install && yarn android-clean && yarn ios-clean && yarn pod-install",
```

Comment:
Found this cool repo https://github.com/pmadruga/react-native-clean-project

can change the clean command to this and it becomes interactive
```
   "clean": "react-native-clean-project",
```

ok to leave the old one if you want too, but this clean project works pretty well.